src/core_plugins/kbn_vislib_vis_types/public/pie.js

```diff
@@ -21,6 +21,7 @@ export default function HistogramVisType(Private) {
         addLegend: true,
         legendPosition: 'right',
         isDonut: true,
+        showTotal: true,
```

Comment:
Please make the default `false`, so we don't change behavior on existing charts.